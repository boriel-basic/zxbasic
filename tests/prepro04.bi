REM LINE 1
#ifdef __MEMORY_CHECK__
REM LINE 3
#else
REM LINE 5
#endif
REM LINE 7

