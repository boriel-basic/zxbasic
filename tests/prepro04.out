
#line 5 "prepro04.bi"

#line 7 "prepro04.bi"


